sale_variant_configurator/views/sale_view.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_order_form">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="priority" eval="20" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree" position="attributes">
+                <attribute name="editable" />
+            </xpath>
+            <xpath
+                expr="//field[@name='order_line']/tree//field[@name='product_id']"
+                position="before"
+            >
+                <field name="product_tmpl_id" string="Product Template" />
+            </xpath>
+            <xpath expr="//field[@name='order_line']" position="attributes">
+                <attribute name="options">{'reload_on_button': true}</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='order_line']/form//field[@name='product_id']"
+                position="attributes"
+            >
+                <attribute
+                    name="attrs"
+                >{'readonly': [('product_updatable', '=', False)]}</attribute>
+            </xpath>
```

Comment:
```suggestion
                <attribute name="readonly">not product_updatable</attribute>
            </xpath>
```
All these changes are necessary according to the migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0 right?